xdsl/dialects/printf.py

```diff
@@ -85,9 +87,29 @@ def parse(cls: type[PrintFormatOp], parser: Parser) -> PrintFormatOp:
         return op
 
 
+@irdl_op_definition
+class PrintCharOp(IRDLOperation):
+    """
+    Print a single character
+
+    Equivalent to putchar in C.
+    """
+
+    name = "printf.print_char"
+    char: Operand = operand_def()
+
+
+@irdl_op_definition
+class PrintIntOp(IRDLOperation):
+    """
+    Print a single Integer
+    """
+
+    name = "printf.print_int"
+    int: Operand = operand_def(builtin.IntegerType)
+
+
 Printf = Dialect(
-    [
-        PrintFormatOp,
-    ],
+    [PrintFormatOp, PrintCharOp, PrintIntOp],
```

Comment:
```suggestion
    [PrintFormatOp, PrintCharOp, PrintIntOp,],
```

Reply:
This would be a useful change to minimise spurious git diffs in the future

Reply:
plz

Reply:
Sure, but what do you mean with 
> to minimise spurious git diffs in the future

?